src/grpc_client.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2015 gRPC authors.
```

Comment:
Dont have a strong opinion here, but wondering if the client code should be in the scripts dir instead

Reply:
I did think about this, the client and server do depend on the generated gRPC python stub classes. If this was a Java project I would extract the common classes into a separate module and the client & server would depend on that module.  I am not familiar with Python best practice. Any guidance here would be appreciated.

Reply:
ok we can take this up in a separate PR

Reply:
I opened an issue to track this: https://github.com/kruize/hpo/issues/37